src/service/follower.proto

```diff
@@ -89,4 +100,6 @@ service follower {
   rpc subnetwork_last_reward_height(
       follower_subnetwork_last_reward_height_req_v1)
       returns (follower_subnetwork_last_reward_height_resp_v1);
+  rpc oui_balance_stream(follower_oui_balance_update_req_v1)
```

Comment:
1. What's the bootstrap here? How do you get the initial balance? Streamed at first connect? for all router keys and their oui?
2. What step do we need the dependency in again vs getting it from the SOL chain?

Reply:
1. The first message from the follower gives you the balance of all of the oui
2. It's for checking the balance to determine if a packet is valid. It would get replace outright with checking the balance on the SOL chain

Reply:
> * The first message from the follower gives you the balance of all of the oui

Gotcha, ok

> * It's for checking the balance to determine if a packet is valid. It would get replace outright with checking the balance on the SOL chain

Good for now. Just put it in a function we can easily replaces  with a solana-sdk one